jobs/cronjob-prune-projects.yaml

```diff
@@ -132,7 +132,7 @@ parameters:
 - name: PROJECT_EXCLUDE_SYSTEM
   displayName: System projects to exclude from the Prune Job
   description: System projects that should not be deleted
-  value: default kube-public kube-service-catalog kube-system management-infra openshift openshift-ansible-service-broker openshift-infra openshift-logging openshift-node openshift-sdn openshift-template-service-broker openshift-web-console openshift-console openshift-metrics-server openshift-monitoring openshift-apiserver openshift-apiserver-operator openshift-authentication openshift-authentication-operator openshift-cluster-machine-approver openshift-cluster-node-tuning-operator openshift-cluster-samples-operator openshift-cluster-storage-operator openshift-cluster-version openshift-config openshift-config-managed openshift-console-operator openshift-controller-manager openshift-controller-manager-operator openshift-dns openshift-dns-operator openshift-etcd openshift-image-registry openshift-infra openshift-ingress openshift-ingress-operator openshift-kube-apiserver openshift-kube-apiserver-operator openshift-kube-controller-manager openshift-kube-controller-manager-operator openshift-kube-scheduler openshift-kube-scheduler-operator openshift-machine-api openshift-machine-config-operator openshift-marketplace openshift-multus openshift-network-operator openshift-node openshift-operator-lifecycle-manager openshift-operators openshift-sdn openshift-service-ca openshift-service-ca-operator openshift-service-catalog-apiserver-operator openshift-service-catalog-controller-manager-operator
+  value: management-infra istio-system cluster-ops
```

Comment:
this new `value` doesn't quite make sense considering that line 130 is `cluster-maintenance` ... and including `istio` seems a bit random. Perhaps we can leave this new value at `cluster-maintenance` ?

Reply:
@oybed thanks, I hadn't caught that one... the project switches between `cluster-ops` and `cluster-maintenance in several places I think... I'm for keeping it consistent but it seems there may be some debate over it? "ops" is shorter but "maintenance" is perhaps more clear...

```
$ grep -R cluster-maintenance
charts/cronjob-aws-ocp-snap/values.yaml:namespace: cluster-maintenance
charts/cronjob-prune-projects/values.yaml:namespace: cluster-maintenance
jobs/cronjob-prune-projects.yaml:  value: "cluster-maintenance"
jobs/cronjob-aws-ocp-snap.yaml:  value: "cluster-maintenance"
$ grep -R cluster-ops
charts/cronjob-ldap-group-sync-secure/values.yaml:namespace: openshift-cluster-ops
charts/cronjob-prune-images/values.yaml:namespace: cluster-ops
charts/cronjob-ldap-group-sync/values.yaml:namespace: openshift-cluster-ops
charts/cronjob-prune-builds-deployments/values.yaml:namespace: cluster-ops
jobs/cronjob-prune-builds-deployments.yml:  value: "cluster-ops"
jobs/cronjob-ldap-group-sync.yml:    value: "openshift-cluster-ops"
jobs/cronjob-prune-images.yml:  value: "cluster-ops"
jobs/cronjob-ldap-group-sync-secure.yml:  value: "openshift-cluster-ops"
```

`management-infra` I think was a leftover from some optional 3.11 stuff (it was already in this list).  `istio-system` is certainly desirable if one installs service mesh (we had) but we could remove it I suppose.  We'd run the risk of people not reviewing this variable carefully and borking up their cluster (happened to me). I added it here because it is not specifically addressed in the bash script and it is an optional piece to the cluster. If they don't have it, it shouldn't hurt them anyways.  Basically I read this variable as a place we should be putting any additional namespaces that we _know_ might be installed at any time but that are not a de-facto default OCP install (those are in the bash script).